Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -23,6 +23,9 @@ struct PlaceUtilities {
 
     static func filterPlacesForCarousel(_ places: [Place]) -> [Place] {
         return places.filter { place in
+            // always show places if they have events
+            guard place.events.isEmpty else { return true }
```

Comment:
Are these events guaranteed to be soon enough that we want to show them? Or is this just for testing? If just testing, it could be good to add a comment to that effect.